// gd-com/utils/dist/get/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.getVar = exports.getDouble = exports.getFloat = exports.getString = exports.getU64 = exports.getU32 = exports.getU16 = exports.getU8 = exports.get64 = exports.get32 = exports.get16 = exports.get8 = void 0;
var get_8_1 = require("./get_8");
Object.defineProperty(exports, "get8", { enumerable: true, get: function () { return get_8_1.get8; } });
var get_16_1 = require("./get_16");
Object.defineProperty(exports, "get16", { enumerable: true, get: function () { return get_16_1.get16; } });
var get_32_1 = require("./get_32");
Object.defineProperty(exports, "get32", { enumerable: true, get: function () { return get_32_1.get32; } });
var get_64_1 = require("./get_64");
Object.defineProperty(exports, "get64", { enumerable: true, get: function () { return get_64_1.get64; } });
var get_u8_1 = require("./get_u8");
Object.defineProperty(exports, "getU8", { enumerable: true, get: function () { return get_u8_1.getU8; } });
var get_u16_1 = require("./get_u16");
Object.defineProperty(exports, "getU16", { enumerable: true, get: function () { return get_u16_1.getU16; } });
var get_u32_1 = require("./get_u32");
Object.defineProperty(exports, "getU32", { enumerable: true, get: function () { return get_u32_1.getU32; } });
var get_u64_1 = require("./get_u64");
Object.defineProperty(exports, "getU64", { enumerable: true, get: function () { return get_u64_1.getU64; } });
var get_string_1 = require("./get_string");
Object.defineProperty(exports, "getString", { enumerable: true, get: function () { return get_string_1.getString; } });
var get_float_1 = require("./get_float");
Object.defineProperty(exports, "getFloat", { enumerable: true, get: function () { return get_float_1.getFloat; } });
var get_double_1 = require("./get_double");
Object.defineProperty(exports, "getDouble", { enumerable: true, get: function () { return get_double_1.getDouble; } });
var get_var_1 = require("./get_var");
Object.defineProperty(exports, "getVar", { enumerable: true, get: function () { return get_var_1.getVar; } });
