// gd-com/utils/dist/constants.js
"use strict";
// take a look = http=//docs.godotengine.org/en/latest/tutorials/misc/binary_serialization_api.html
Object.defineProperty(exports, "__esModule", { value: true });
exports.TYPE = void 0;
var TYPE;
(function (TYPE) {
    TYPE[TYPE["NULL"] = 0] = "NULL";
    TYPE[TYPE["BOOL"] = 1] = "BOOL";
    TYPE[TYPE["INTEGER"] = 2] = "INTEGER";
    TYPE[TYPE["FLOAT"] = 3] = "FLOAT";
    TYPE[TYPE["STRING"] = 4] = "STRING";
    TYPE[TYPE["VECTOR2"] = 5] = "VECTOR2";
    TYPE[TYPE["RECT2"] = 6] = "RECT2";
    TYPE[TYPE["VECTOR3"] = 7] = "VECTOR3";
    TYPE[TYPE["TRANSFORM2D"] = 8] = "TRANSFORM2D";
    TYPE[TYPE["PLANE"] = 9] = "PLANE";
    TYPE[TYPE["QUAT"] = 10] = "QUAT";
    TYPE[TYPE["AABB"] = 11] = "AABB";
    TYPE[TYPE["BASIS"] = 12] = "BASIS";
    TYPE[TYPE["TRANSFORM"] = 13] = "TRANSFORM";
    TYPE[TYPE["COLOR"] = 14] = "COLOR";
    TYPE[TYPE["NODE_PATH"] = 15] = "NODE_PATH";
    TYPE[TYPE["RID"] = 16] = "RID";
    TYPE[TYPE["OBJECT"] = 17] = "OBJECT";
    TYPE[TYPE["DICTIONARY"] = 18] = "DICTIONARY";
    TYPE[TYPE["ARRAY"] = 19] = "ARRAY";
    TYPE[TYPE["RAW_ARRAY"] = 20] = "RAW_ARRAY";
    TYPE[TYPE["INT_ARRAY"] = 21] = "INT_ARRAY";
    TYPE[TYPE["REAL_ARRAY"] = 22] = "REAL_ARRAY";
    TYPE[TYPE["STRING_ARRAY"] = 23] = "STRING_ARRAY";
    TYPE[TYPE["VECTOR2_ARRAY"] = 24] = "VECTOR2_ARRAY";
    TYPE[TYPE["VECTOR3_ARRAY"] = 25] = "VECTOR3_ARRAY";
    TYPE[TYPE["COLOR_ARRAY"] = 26] = "COLOR_ARRAY";
    // MAX= 27
})(TYPE = exports.TYPE || (exports.TYPE = {}));
