// gd-com/utils/dist/types/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.GodotVector3 = exports.GodotVector2 = exports.GodotTransform2D = exports.GodotTransform = exports.GodotRect2 = exports.GodotQuat = exports.GodotPlane = exports.GodotColor = exports.GodotBasis = exports.GodotAabb = void 0;
var GodotAabb_1 = require("./GodotAabb");
Object.defineProperty(exports, "GodotAabb", { enumerable: true, get: function () { return GodotAabb_1.GodotAabb; } });
var GodotBasis_1 = require("./GodotBasis");
Object.defineProperty(exports, "GodotBasis", { enumerable: true, get: function () { return GodotBasis_1.GodotBasis; } });
var GodotColor_1 = require("./GodotColor");
Object.defineProperty(exports, "GodotColor", { enumerable: true, get: function () { return GodotColor_1.GodotColor; } });
var GodotPlane_1 = require("./GodotPlane");
Object.defineProperty(exports, "GodotPlane", { enumerable: true, get: function () { return GodotPlane_1.GodotPlane; } });
var GodotQuat_1 = require("./GodotQuat");
Object.defineProperty(exports, "GodotQuat", { enumerable: true, get: function () { return GodotQuat_1.GodotQuat; } });
var GodotRect2_1 = require("./GodotRect2");
Object.defineProperty(exports, "GodotRect2", { enumerable: true, get: function () { return GodotRect2_1.GodotRect2; } });
var GodotTransform_1 = require("./GodotTransform");
Object.defineProperty(exports, "GodotTransform", { enumerable: true, get: function () { return GodotTransform_1.GodotTransform; } });
var GodotTransform2D_1 = require("./GodotTransform2D");
Object.defineProperty(exports, "GodotTransform2D", { enumerable: true, get: function () { return GodotTransform2D_1.GodotTransform2D; } });
var GodotVector2_1 = require("./GodotVector2");
Object.defineProperty(exports, "GodotVector2", { enumerable: true, get: function () { return GodotVector2_1.GodotVector2; } });
var GodotVector3_1 = require("./GodotVector3");
Object.defineProperty(exports, "GodotVector3", { enumerable: true, get: function () { return GodotVector3_1.GodotVector3; } });
